identity-wallet/bindings/EventType.ts

```diff
@@ -0,0 +1,3 @@
+// This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+
+export type EventType = "connection_added" | "credentials_added" | "credentials_shared";
```

Comment:
I think `"ConnectionAdded"` is more conventional than `"connection_added"` for the frontend.